pkg/webui/console/containers/notifications/notifications.styl

```diff
@@ -0,0 +1,153 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.notifications-container
+  border-radius: $br.s
+  min-height: 85vh
+  +media-query($bp.s)
+    min-height: 100%
+
+.notification-list
+  display: flex
+  flex-direction: column
+  border-right: 1px solid #E4E4E4
+  border-radius: $br.m 0 0 $br.m
+  background-color: white
+  padding: 0 !important
+
+  +media-query($bp.s)
+    margin-top: 0 !important
+    margin-bottom: 0 !important
+
+  &-change-button
+    color: $c-active-blue !important
+
+.notification-header
+  border-bottom: 1px solid #E4E4E4
+  align-items: center
+  padding: $cs.s 0
+
+  +media-query($bp.s)
+    display: flex
+
+  &-icon
+    top: 5px !important
+    padding-right: $cs.xs
+
+    &.notifications
+      color: $c-active-blue
+
+    &.archived
+      color: $c-warning
+
+  &-date
+    font-size: 1.1rem
+    padding-right: $cs.xs
+    padding-left: 0
+    padding-top: 0.2rem
+
+  .total-notifications
+    border-radius: 45%
+    background-color: $c-active-blue
+    color: white
+    padding: 0 0.8rem
+    margin-left: $cs.xs
+    line-height: 1.6
+    position: relative
+    top: 5px
+    height: 70%
+
+  &-action
+    display: flex
+    justify-content: end
+    align-items: center
+    +media-query($bp.s)
+      max-width: fit-content !important
+
+  &-title
+    +media-query($bp.s)
+      max-width: 70% !important
+
+  .back-button
+    height: 100%
+
+    .icon
+      font-size: 3rem
+
+.notification-preview
+  width: 100%
+  position: relative
+  border: none !important
+  border-bottom: 1px solid #E4E4E4 !important
+  border-radius: 0 !important
+  box-shadow: none !important
+  min-height: 7rem
+  padding: $cs.l !important
+  justify-content: start !important
+  flex-direction: column !important
+  align-items: start !important
```

Comment:
This is a lot of `!important` here. I don't want to be too strict about it but it is definitely a code smell. Any chance you can resolve this by using better selector specificity?